### PR TITLE
Add model credentials to expanded row in profile.

### LINKED ISSUE #2338: view credentials via juju-gui
It would be nice if the gui displayed what credentials were being used for the current model. 
screen shot 2017-01-19 at 7 00 38 am

### PR DESCRIPTION
**Pull request by kadams54**
Fixes #2338.

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7056/

**Comment by kadams54** [comment]
## QA

1. Fire up `make run` & guiproxy.
2. Head to your profile.
3. Verify that models display correctly, both in their unexpanded and expanded states.
4. Verify that the credentials display in the expanded state.
5. Verify that charms and bundles also display correctly (i.e., no regressions due to the changes in models). If you don't have any on your own profile, see `/u/frankban`.

File: jujugui/static/gui/src/app/components/user-profile/entity/entity.js [hunk 1]
```diff
@@ -241,19 +241,42 @@ YUI.add('user-profile-entity', function() {
...
+      if (this.props.type !== 'model') {
+        return null;
       }
+      const model = this.props.entity;
+      // See the _generateRow function in model-list.js for matching logic.
+      // Both sections should be kept roughly in sync.
+      const region = model.region || 'no region';
+      let owner = '--';
+      if (model.owner) {
+        owner = model.ownder = model.owner.split('@')[0];
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/entity/entity.js]: typo, `model.owner`

Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/entity/entity.js]: I don't like how a UI component is modifying the value of the owner for the rest of the app. Was it intentional to remove the domain for the owner for everywhere in the app?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7059/

**Comment by hatched** [comment]
Thanks @kadams54 this looks awesome. QA OK

**Comment by kadams54** [comment]
Thanks! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7060/

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2227

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**